hbase-server/src/main/java/org/apache/hadoop/hbase/master/ServerManager.java

```diff
@@ -715,7 +716,10 @@ public static void closeRegionSilentlyAndWait(AsyncClusterConnection connection,
           return;
         }
       } catch (IOException ioe) {
-        if (ioe instanceof NotServingRegionException) {
+        if (ioe instanceof NotServingRegionException ||
+          (ioe instanceof RemoteWithExtrasException &&
+            ((RemoteWithExtrasException)ioe).unwrapRemoteException()
+              instanceof NotServingRegionException)) {
```

Comment:
This change is related to this PR?

Reply:
Yes, it is. The excess meta replicas are not assigned so when we try to close them a NotServingRegionException is thrown. Unfortunately it gets wrapped in a RemoteWithExtrasException which was not handled so the logic kept trying to close the region not realizing it was not necessary.

Reply:
Sounds good, I thought it was specific to unit test and only in the case of unit test (based on config), this is happening but yeah if this is generic, we are good.
Thanks @BukrosSzabolcs